chart/templates/timescaledb-grafana-datasource.yaml

```diff
@@ -1,6 +1,4 @@
 {{- $kubePrometheus := index .Values "kube-prometheus-stack" -}}
-{{- $tsdb := index .Values "timescaledb-single" -}}
-{{ if and $tsdb.enabled $kubePrometheus.grafana.enabled ((($kubePrometheus.grafana).timescale).datasource).enabled -}}
```

Comment:
I think some of those checks are still valid. For example if user is not enabling grafana, we don't need to provision a datasource user.

Reply:
ah yeah, I forgot to add this back.  Thanks!